src/resources/scepter.ts

```diff
@@ -140,7 +152,7 @@ const SKILL_OPTIONS: ScepterSkill[] = [
 let bestScepterSkills: ScepterSkill[] | null = null;
 function getBestScepterSkills(): ScepterSkill[] {
   return (bestScepterSkills ??= SKILL_OPTIONS.filter(
-    ({ skill }) => AugustScepter.todaysSkill() !== skill,
+    ({ skill, type }) => AugustScepter.todaysSkill() !== skill && type !== "fight",
```

Comment:
I understand what you're doing here, I'm just not sure if I agree. If the best skill to cast is a fight, let them fight!